bundles/binding/org.openhab.binding.modbus/src/main/java/org/openhab/binding/modbus/internal/ModbusSlave.java

```diff
@@ -182,9 +185,8 @@ protected static boolean translateCommand2Boolean(Command command) {
      * @throws ModbusException ModbusIOException on IO errors, ModbusSlaveException with protocol level exceptions
      * @throws ModbusUnexpectedTransactionIdException when response transaction id does not match the request
      */
-    private void setCoil(Command command, ModbusBindingConfig config)
+    private void setCoil(Command command, ModbusBindingConfig config, int readRegister, int writeRegister)

```

Comment:
why the additional parameters `readRegister` and `writeRegister`? Shouldn't config include that?